.schema/config.schema.json

```
@@ -355,7 +355,9 @@
       }
     },
     "dsn": {
-      "type": "string"
+      "type": "string",
+      "title": "Data Source Name",
+      "description": "DSN is used to specify the database credentials as a connection URI."
```

Can you add your example here? It should then be rendered into the markdown file.
```suggestion
      "description": "DSN is used to specify the database credentials as a connection URI.",
      "examples": ["postgres://user:password@localhost:5432/database"]
```
To regenerate the configuration page run (in the project's root directory)
```
$ node docs/scripts/config.js docs/config.js
```
and then commit the markdown aswell.
We should really document how to do that.

---

Okay, did that. I also added a quick instruction for how to do it in the CONTRIBUTING.md. Maybe add it to the PR checks?